runtime/jvmti/jvmtiCapability.c

```diff
@@ -378,6 +378,13 @@ jvmtiAddCapabilities(jvmtiEnv* env,
 			 * Set it to 512KB which is default sampling interval as per JEP 331 specification.
 			 */
 			vm->memoryManagerFunctions->j9gc_set_allocation_sampling_interval(currentThread, 512 * 1024);
+			
+			if (jvmtiData->flags & J9JVMTI_FLAG_SAMPLED_OBJECT_ALLOC_ENABLED) {
```

Comment:
Can you make this a boolean comparison?  either `((jvmtiData->flags & J9JVMTI_FLAG_SAMPLED_OBJECT_ALLOC_ENABLED) == J9JVMTI_FLAG_SAMPLED_OBJECT_ALLOC_ENABLED` or use the `J9_ARE_ANY_BITS_SET(jvmtiData->flags, J9JVMTI_FLAG_SAMPLED_OBJECT_ALLOC_ENABLED)` macro

Reply:
Changed to `J9_ARE_ANY_BITS_SET(jvmtiData->flags, J9JVMTI_FLAG_SAMPLED_OBJECT_ALLOC_ENABLED)` macro.